RecoTracker/LSTCore/src/alpaka/Quintuplet.h

```diff
@@ -1694,11 +1694,31 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
                                   TripletsOccupancyConst tripletsOccupancy,
                                   Quintuplets quintuplets,
                                   QuintupletsOccupancy quintupletsOccupancy,
-                                  ObjectRangesConst ranges,
+                                  ObjectRanges ranges,
```

Comment:
Is this change left over from previously using the object ranges struct to store the pairs? Or do you changes the ranges struct in the kernel somewhere.

Reply:
You are right, I don't need to have a variant object range and I changed back